example/src/__snapshots__/App.test.jsx.snap

```diff
@@ -104,5 +104,11 @@ exports[`Basic test should render 1`] = `
   >
     I'm aligned right, but left in RTL.
   </p>
+  <h2>
+    Exposed Paragon version
+  </h2>
+  <p>
+    "\\"20.39.1\\""
+  </p>
```

Comment:
I think this snapshot is going to break quite often. Might make sense to keep it static for tests.

Reply:
Yeah, I agree haha. Will need to find another way to test.